Packs/CortexCloudSec/Integrations/AWS/AWS.py

```diff
@@ -0,0 +1,198 @@
+            'BlockPublicAcls': public_access_block_configuration.get('BlockPublicAcls'),
+            'IgnorePublicAcls': public_access_block_configuration.get('IgnorePublicAcls'),
+            'BlockPublicPolicy': public_access_block_configuration.get('BlockPublicPolicy'),
+            'RestrictPublicBuckets': public_access_block_configuration.get('RestrictPublicBuckets')
+        }
+    except client_session.exceptions.NoSuchEntityException:
+        kwargs = {}
+        
+    if args.get('block_public_acls'):
+        kwargs.update({ 'BlockPublicAcls': argToBoolean(args.get('block_public_acls')) })
+    if args.get('ignore_public_acls'):
+        kwargs.update({ 'IgnorePublicAcls': argToBoolean(args.get('ignore_public_acls')) })
+    if args.get('block_public_policy'):
+        kwargs.update({ 'BlockPublicPolicy': argToBoolean(args.get('block_public_policy')) })
+    if args.get('restrict_public_buckets'):
+        kwargs.update({ 'RestrictPublicBuckets': argToBoolean(args.get('restrict_public_buckets')) })
+
+
+    response = client_session.put_public_access_block(Bucket=args.get('bucket'),
+                                                      PublicAccessBlockConfiguration=kwargs)
+
+    if response['ResponseMetadata']['HTTPStatusCode'] == HTTPStatus.OK:
+        return CommandResults(
+            readable_output=f"Successfully applied public access block to the {args.get('bucket')} bucket")
+    return CommandResults(readable_output=f"Couldn't apply public access block to the {args.get('bucket')} bucket")
+
+# =================== #
+# IAM commands
+# =================== #
+
+def get_account_password_policy(aws_client: AWSClient, args: Dict[str, Any]) -> CommandResults:
+    client_session = aws_client.aws_session(service='iam', region=args.get('region'))
+    response = client_session.get_account_password_policy()
+    data = json.loads(json.dumps(response['PasswordPolicy'], cls=DatetimeEncoder))
+    data.update({'AccountId': args.get('account_id')})
+    
+    human_readable = tableToMarkdown('AWS IAM Account Password Policy', data)
+    
+    return CommandResults(outputs=data, readable_output=human_readable, outputs_prefix='AWS.IAM.PasswordPolicy',
+                          outputs_key_field='AccountId')
+    
+def update_account_password_policy(aws_client: AWSClient, args: Dict[str, Any]) -> CommandResults:
+    client_session = aws_client.aws_session(service='iam', region=args.get('region'))
+    try:
+        response = client_session.get_account_password_policy()
+        kwargs = response['PasswordPolicy']
+    except client_session.exceptions.NoSuchEntityException:
+        kwargs = {}
+    # ExpirePasswords is part of the response but cannot be included
+    # in the request
+    if 'ExpirePasswords' in kwargs:
+        kwargs.pop('ExpirePasswords')
+    if args.get('minimum_password_length'):
+        kwargs.update({'MinimumPasswordLength': int(args.get('minimum_password_length'))})
+    if args.get('require_symbols'):
```

Comment:
```suggestion
    if argToBoolean(args.get('require_symbols')):
```
Same for all Boolean arguments